src/js/tech/tech.js

```diff
@@ -127,33 +129,12 @@ class Tech extends Component {
     this.duration_ = this.duration();
   }
 
-  bufferedPercent() {
-    let bufferedDuration = 0,
-        start, end;
-
-    if (!this.duration_) {
-      return 0;
-    }
-
-    let buffered = this.buffered();
-
-    if (!buffered || !buffered.length) {
-      buffered = createTimeRange(0,0);
-    }
-
-    for (var i=0; i<buffered.length; i++){
-      start = buffered.start(i);
-      end   = buffered.end(i);
-
-      // buffered end can be bigger than duration by a very small fraction
-      if (end > this.duration_) {
-        end = this.duration_;
-      }
-
-      bufferedDuration += end - start;
-    }
+  buffered() {
+    return null;
```

Comment:
If this isn't used outside of tests do we need to define it here? Should we change something about the test or whatever relies on buffered?

Otherwise I think we should follow the spec. 
> The buffered attribute must return a new static normalised TimeRanges object that represents the ranges of the media resource, if any, that the user agent has buffered, at the time the attribute is evaluated. Users agents must accurately determine the ranges available, even for media streams where this can only be determined by tedious inspection.

http://www.w3.org/html/wg/drafts/html/master/semantics.html#dom-media-buffered